### PR TITLE
Collection detail controls horizontal alignment

### LINKED ISSUE #1081: Collection detail controls horizontal alignment
Make order 
- filter
- search (probably will be smaller, but expanded)
- BUY NOW
- Layout change
- Pagination

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the KodaDot NFT gallery.

👇 Let's do a quick check before the merge.

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1081
Have added a global state for layout class, as I have made a common component so that it could be easily integrated into collection horizontal alignment 

### Had issue bounty label ?
- [x] Fill up your KSM address: 
[Payout](https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot for others, is best to understand changes
Screenshot 2021-11-11 at 7 52 41 PM
.

### CODE REVIEW

**Comment by yangwao** [comment]
Great stuff! 
image

File: src/components/rmrk/Gallery/GalleryCardList.vue [hunk 1]
```diff
@@ -44,17 +29,21 @@ import { Component, Prop, Vue } from 'vue-property-decorator'
...
 }
 
 @Component({ components })
 export default class GalleryCardList extends Vue {
   @Prop({ default: 'nftDetail' }) public type!: string;
   @Prop({ default: 'rmrk/detail' }) public link!: string;
   @Prop() public items!: RmrkType[];
-
-  protected layout = 'is-one-third-desktop is-one-third-tablet'
+  @Prop({ default: false }) public horizontalLayout!: boolean;
```
Comment (vikiival) [inline src/components/rmrk/Gallery/GalleryCardList.vue:41]: missing new line after

File: src/components/rmrk/Gallery/GalleryCardList.vue [hunk 2]
```diff
@@ -44,17 +29,21 @@ import { Component, Prop, Vue } from 'vue-property-decorator'
...
+  get classLayout() {
```
Comment (vikiival) [inline src/components/rmrk/Gallery/GalleryCardList.vue:42]: missing return type

**Review by vikiival — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
Payout sent! https://kusama.subscan.io/extrinsic/0x871eba0643e95968db8969b22917df63c60e2a036a94bc1d49f0fab20bc21493

**Pull request merged**